pfb-analysis/scripts/entrypoint.sh

```diff
@@ -57,5 +57,5 @@ EOF
 bash
 
 # shutdown postgres
-kill $POSTGRES_PROC
+su postgres -c "/usr/lib/postgresql/9.6/bin/pg_ctl stop"
```

Comment:
Assuming this is just so that the server process is shut down cleanly? Were you running into issues with the previous implementation?